Automattic/vip-go-mu-plugins
# Update Jetpack 9.4 subtree to the final release

## Description

Update subtree `jetpack-9.4` to contain the released Jetpack 9.4 version

## Changelog Description

### Plugin Updated: Jetpack 9.4

We have upgraded the Jetpack 9.4 subtree to contain the final Jetpack 9.4 release. This will be set as the default for all unpinned sites on Thursday 4th.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Add `define ( 'VIP_JETPACK_PINNED_VERSION', '9.4' );`
1. Go to the Jetpack dashboard in `wp-admin`
1. Ensure that the active version is 9.4 (not 9.3.1, not 9.4-beta)
1. Ensure everything keeps working
